tests/DependencyInjection/KnpMenuExtensionTest.php

```diff
@@ -72,8 +70,7 @@ public function testDisableContainerAwareProvider()
     {
         $container = new ContainerBuilder();
         $loader = new KnpMenuExtension();
-        $loader->load([['providers' => ['container_aware' => false]]], $container);
+        $loader->load([['providers' => []]], $container);
```

Comment:
this legacy test should be removed entirely instead, as it is the test about disabling the ContainerAwareProvider